dockerfiles/Dockerfile.cross

```diff
@@ -1,3 +1,3 @@
-FROM    dockercore/golang-cross:1.9.4@sha256:b8d43ef11ccaa15bec63a1f1fd0c28a0e729074aa62fcfa51f0a5888f3571315
+FROM    dockercore/golang-cross:latest@sha256:9e626f4ed30718deb2f8dd6b36c454ff11cbb5c9900f971ebfe37582fd0d5a72
```

Comment:
Didn't have push access to the repo, so couldn't tag it as `1.9.5` 😊

Reply:
Done, `1.9.5` pushed 😉